src/cc1101.cpp

```diff
@@ -46,17 +51,22 @@ const uint8_t cc1101::initVal[] PROGMEM =
 //0x56, // 21 FREND1    56     RX filter bandwidth = 101 kHz, FREND1 = 0x56
 0xB6, // 21 FREND1    B6     RX filter bandwidth > 101 kHz, FREND1 = 0xB6
 0x11, // 22 FREND0    16     0x11 for no PA ramping
-0xE9, // 23 FSCAL3    A9    E9 ??
+0xE9, // 23 FSCAL3    A9     E9 ??
 0x2A, // 24 FSCAL2    0A
-0x00, // 25 FSCAL1    20    19 ??
+0x00, // 25 FSCAL1    20     19 ??
 0x1F, // 26 FSCAL0    0D
 0x41, // 27 RCCTRL1
 0x00, // 28 RCCTRL0
 };
 
+void MSG_PRINTtoHEX(uint8_t a) { // this function is the alternative to sprintf(b, "%02x", xxx(i))
```

Comment:
Ich war jetzt ein wenig am hin und her überlegen, ansich hat diese Funktion ja nichts mit dem cc1101 zu tun.
Vielleicht ist die besser in output.h aufgehoben?

Reply:
Die Funktion in output.h wäre fast der gleiche Fall als diese in der output.cpp war. Diese Datei wurde aufgelöst weil es Probleme mit deinen Tests gab. In eine Header Datei die Funktion einzubauen gehört sich ja "eigendlich" nicht wenn wir Header und cpp Gesamt betrachten.

Wenn die funktion in der Datei definiert wird wo diese benötigt wird, so nimmt diese den gleichen Speicher ein, als wenn diese ausgelagert wird. Sollten wir nun wieder zu einer separaten Datei für diese Funktion pändeln, so kommen vermutlich wieder Abhängigkeiten ans Licht welche nicht erfüllt werden können.

Ja, wir können dies versuchen aber genau aus diesem Grund, weil Abhängigkeiten nicht erfüllt wurden und Variablen nicht übernommen wurden, so wurde die CPP aufgelöst weil spätestens deine Tests als separate Bibliothek dann fehl schlugen.

Sollte es dein wunsch sein, so werde ich versuchen diese in der H Datei zu positionieren.